eclass/usr-ldscript.eclass

```diff
@@ -35,6 +35,7 @@ IUSE="split-usr"
 # correctly to point to the latest version of the library present.
 gen_usr_ldscript() {
 	local lib libdir=$(get_libdir) output_format="" auto=false suffix=$(get_libname)
+	local prefix=$(usex prefix-guest "${EPREFIX}" "")
```

Comment:
I feel rather silly, as I've just noticed the `use prefix && return` right below. :sweat_smile: I decided to keep this in anyway, just in case that changes. Comment added.